llvm/lib/IR/FPAccuracy.cpp

```diff
@@ -0,0 +1,133 @@
+//===-- FPAccuracy.cpp ---- FP Accuracy Support ---------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+/// @file
+/// This file contains the implementations of functions that map standard
+/// accuracy levels to required accuracy expressed in terms of ulps.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/IR/FPAccuracy.h"
+#include "llvm/IR/Instruction.h"
+#include "llvm/IR/IntrinsicInst.h"
+#include "llvm/IR/Intrinsics.h"
+#include <optional>
```

Comment:
I don't see any uses of `std::optional` in the file, am I missing something? What is the reason to include the header?